lib/heap_sort.rb

```diff
@@ -4,5 +4,15 @@
 # Time Complexity:  ?
 # Space Complexity: ?
 def heap_sort(list)
-  raise NotImplementedError, "Method not implemented yet..."
+  heap = MinHeap.new
+  
+  list.each do |elem|
+    heap.add(elem)
+  end
+  
+  sorted = []
+  heap.each do |elem|
```

Comment:
Heaps don't have a .each method.  

```suggestion
  while !heap.empty?
```